registrations/management/commands/report_pmtct_risks.py

```diff
@@ -103,8 +104,14 @@ def add_to_result(risk, reg):
                     validated=True)
 
                 for registration in registrations:
+                    mom_dob = registration["data"].get("mom_dob")
+                    if not mom_dob:
+                        identity = self.get_identity(
+                            ids_client, registration["registrant_id"])
+                        mom_dob = identity["details"].get("mom_dob")
```

Comment:
In that case, `.get` returns `None`. I think the code downstream though might have an issue being passed `None`, should I maybe add a check and skip processing the registration if that's the case?